lib/bisu.rb

```diff
@@ -33,6 +33,9 @@ def run(options)
     end
 
     Bisu::Logger.print_summary
+    if (options[:strict] && Bisu::Logger.summary[:warn] > 0)
+      Logger.error("Found a warning while in strict mode")
```

Comment:
How come you're saying `Bisu::Logger` sometimes, and sometimes just `Logger`?

Also, I'm surprised that just _logging_ an error, causes the build to fail?

Reply:
Removed the unneeded `Bisu::` references.
Yeah I was surprised about that too, semantically it doesn't seem right :/ but I since I saw the original developers were failing the builds this way, and I wanted a quick way to fail it in case of any warning, I just used it